packages/api/src/routes/medical/internal-cq.ts

```diff
@@ -104,4 +106,22 @@ router.get(
   })
 );
 
+// BELOW ARE THE ROUTES PERTAINING TO THE IHE-GATEWAY
+
+/**
+ * POST /internal/carequality/patient-discovery
+ *
+ * Receives a Patient Discovery request from the IHE Gateway
+ */
+router.post(
+  "/patient-discovery",
```

Comment:
ditto: business name, "create"?

Reply:
Same once comment is addressed

Reply:
why is "create" the business name? we're not creating a patient here

patient discovery (PD) is appropriate here - and thought we were on the same page based on this https://metriport.slack.com/archives/C065BLRBUDQ/p1701200344160359?thread_ts=1701037512.758309&cid=C065BLRBUDQ

Reply:
Yeah, I agree "create" it's not the best name since we're actually searching for patients - in which case, "discovery" seems the most appropriate one for a few reasons.

> thought we were on the same page

Yeah, I thought we were. What does you make you think we're not, just because I suggested the wrong name?

From my understanding, we're giving preference to business operation names instead of tech/protocol oriented (i.e., Doc Query vs ITI-48 - if I got the number right).

Now that we're here, I'm thinking that `POST /patient-discovery` doesn't do what it "sounds" like. One needs explanation that this endpoint doesn't actually do PD, but it's the response of a PD request.

Maybe append "/response"?

`POST /patient-discovery/response`